src/rules/autoregressive/marginals.jl

```diff
@@ -1,9 +1,9 @@
 
-@marginalrule AR(:y_x) (m_y::NormalDistributionsFamily, m_x::NormalDistributionsFamily, q_θ::NormalDistributionsFamily, q_γ::GammaShapeRate, meta::ARMeta) = begin
+@marginalrule AR(:y_x) (m_y::Any, m_x::Any, q_θ::NormalDistributionsFamily, q_γ::Any, meta::ARMeta) = begin
```

Comment:
I'm hesitant of using `::Any` for messages, for marginals it is fine. We can discuss.